languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -8752,3 +8753,106 @@ aliançazinha	aliançazinha	NCFS000
 aliançazinhas	aliançazinha	NCFP000
 descompressor	descompressor	NCMS000
 descompressores	descompressor	NCMP000
+home	home	NCMS000
+homes	home	NCMP000
+PIN	PIN	NCMx000
```

Comment:
Why not tag it as singular? There is a plural form (`PINs`).

Reply:
Typo